src/base/stack.c

```diff
@@ -710,7 +710,7 @@ void xdebug_error_cb(int orig_type, const char *error_filename, const unsigned i
 #ifdef PHP_WIN32
 			if (type==E_CORE_ERROR || type==E_CORE_WARNING) {
 #if PHP_VERSION_ID >= 80000
-				syslog(LOG_ALERT, "PHP %s: %s (%s)", error_type_str, ZSTR_VAL(message), GetCommandLine());
+				php_syslog(LOG_ALERT, "PHP %s: %s (%s)", error_type_str, ZSTR_VAL(message), GetCommandLine());
```

Comment:
This should have an issue (see: https://github.com/xdebug/xdebug/blob/master/CONTRIBUTING.rst#working-on-a-bug-fix), and in its own commit/PR with a test case.

Reply:
Okay, will try to come up with a dedicated issues, and a proper PR for that.